appcontainer/ExceptionDialogModel.js

```diff
@@ -58,16 +58,27 @@ export class ExceptionDialogModel {
     setUserMessage(userMessage) {
         this.userMessage = userMessage;
     }
-    
+
     async sendReportAsync() {
         const {exception, userMessage, options} = this;
 
-        await XH.exceptionHandler.logOnServerAsync({
+        const success = await XH.exceptionHandler.logOnServerAsync({
             exception,
             userMessage,
             userAlerted: true
         });
 
-        if (!options.requireReload) this.close();
+        if (success) {
+            await XH.alert({title: 'Message Sent', message: 'Your error has been reported.'});
```

Comment:
Are we sure we don't want to just toast here? Seems like the user has probably already done enough that we don't need to require another click. Realize thats not symmetrical with the code below, but the case there seems to justify the alert.